scripts/vagrant/multi/README.md

```diff
@@ -0,0 +1,76 @@
+# Vagrant
```

Comment:
nit This is quite similar to the single node setup, can we somehow put them together? May not be worth it right now though